projects/plugins/crm/src/automation/class-base-condition.php

```diff
@@ -66,19 +50,17 @@ abstract class Base_Condition extends Base_Step implements Condition {
 	public function __construct( array $step_data ) {
 		parent::__construct( $step_data );
 
-		$this->next_step_true  = $step_data['next_step_true'] ?? null;
-		$this->next_step_false = $step_data['next_step_false'] ?? null;
-		$this->logger          = Automation_Logger::instance();
+		$this->logger = Automation_Logger::instance();
 	}
 
 	/**
 	 * Get the next step.
 	 *
 	 * @since $$next-version$$
 	 *
-	 * @return int|null The next step data.
+	 * @return string|int|null The next step data.
```

Comment:
Is there more info somewhere on why we allow both `string` and `int`?

Reply:
It's indirectly documented in the example here: pbhBOz-3Wf-p2#comment-4808

There's been some previous discussions about it as well where the argument was used that strings would make it easier for us when we hardcode workflows

Reply:
Thanks!